packages/react-components/react-button/library/src/components/Button/useButtonStyles.styles.ts

```diff
@@ -126,41 +130,47 @@ const useIconBaseClassName = makeResetStyles({
   display: 'inline-flex',
   justifyContent: 'center',
 
-  fontSize: '20px',
-  height: '20px',
-  width: '20px',
+  fontSize: semanticTokens.sizeCtrlIcon,
+  height: semanticTokens.sizeCtrlIcon,
+  width: semanticTokens.sizeCtrlIcon,
 
-  [iconSpacingVar]: tokens.spacingHorizontalSNudge,
+  [iconSpacingVar]: `calc(${semanticTokens.gapInsideCtrlDefault} + ${semanticTokens.paddingCtrlTextside})`,
 });
 
 const useRootStyles = makeStyles({
   // Appearance variations
   outline: {
-    backgroundColor: tokens.colorTransparentBackground,
-
+    backgroundColor: semanticTokens.backgroundCtrlOutlineRest,
+    border: `${semanticTokens.strokewidthCtrlOutlineRest} solid ${semanticTokens.strokeCtrlOnoutlineRest}`,
+    color: semanticTokens.foregroundCtrlOnoutlineRest,
     ':hover': {
-      backgroundColor: tokens.colorTransparentBackgroundHover,
+      backgroundColor: semanticTokens.backgroundCtrlOutlineHover,
```

Comment:
We're adding more semantic overrides to the hover/active base states here, so we need to ensure they are now overridden in any inherited classes.